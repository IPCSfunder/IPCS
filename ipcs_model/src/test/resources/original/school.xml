<?xml version='1.0' encoding='UTF-8'?>
<dataset>
	<school_type SCHOOL_TYPE_OBJID="1" NAME ="HIGH_SCHOOL"/>
	<school SCHOOL_OBJID="1" name = "PUNGOL_PRIMARY_SCHOOL" ADDRESS="PUNGOL" SCHOOL_TYPE_FK="1"/>
</dataset>