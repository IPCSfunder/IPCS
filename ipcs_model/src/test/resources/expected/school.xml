<?xml version='1.0' encoding='UTF-8'?>
<dataset>
	<school_type NAME ="HIGH_SCHOOL"/>
	<school name = "PUNGOL_PRIMARY_SCHOOL" ADDRESS="PUNGOL" SCHOOL_TYPE_FK="1"/>
	<school name = "PUNGOL_HIGH_SCHOOL" ADDRESS="SENGKANG" SCHOOL_TYPE_FK="1"/>
</dataset>